database-plugins/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java

```diff
@@ -123,16 +138,38 @@ private Connection getConnection(Configuration conf) {
     Connection connection;
     try {
       String url = conf.get(DBConfiguration.URL_PROPERTY);
+      if (url.contains("com.google.cloud.sql.mysql.SocketFactory")) {
```

Comment:
Please parse the URI instead of doing string replacement.

Reply:
Also, we shouldn't be matching specifically for google socketfactory. the support should be generic.

Reply:
fixed.